services/src/pro/layers/postgres_layer_db.rs

```diff
@@ -74,7 +76,7 @@ where
                 &id,
                 &layer.name,
                 &layer.description,
-                &layer.workflow,
+                &serde_json::to_value(&layer.workflow).context(error::SerdeJson)?,
```

Comment:
didn't you implement a ToSQL trait for the other types?

Reply:
no, we store complex values as json because we said it'S too much work to try to map the rust structs to SQL